paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerAdapter.kt

```diff
@@ -85,10 +85,12 @@ interface CustomerAdapter {
          * @param context, the Application context
          * @param customerEphemeralKeyProvider, a callback to retrieve the customer id and
          * ephemeral key. The customer ID is used in this adapter to manage the customer's saved
-         * payment methods.
+         * payment methods. Return [CustomerAdapterResult.failure] with a displayMessage if
+         * something went wrong during the retrieval of the ephemeral key.
```

Comment:
I don’t think we need to mention this here. Let’s add it to the `CustomerEphemeralKeyProvider` docs.